_posts/2024-01-10-15.0.0-release.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "Apache Arrow 15.0.0 Release"
+date: "2024-01-10 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 15.0.0 release. This covers
+over 3 months of development work and includes [**XXX resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 14.0.0 release, Curt Hagenlocher, Xuwei Fu, James Duong and Felipe Oliveira Carvalho
+have been invited to be committers.
+Jonathan Keane and Raúl Cumplido have joined the Project Management Committee (PMC).
+
+As per our tradition of rotating the PMC chair once a year
+Andy Grove was elected as the new PMC chair and VP.
+
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+## C Data Interface notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
```

Comment:
@pitrou can you help with the C++ notes?

Reply:
cc @mapleFU @zeroshade @bkietz

Reply:
@wgtmac And me can help finish parquet part this week

Reply:
@raulcd I read through the C++ notes and there are a lot of changes but these seemed worth mentioning:

Add support for external Gandiva function registries (GH-37753).
Implemented the Float16 logical type for C++ and Parquet. (GH-36036).
Added implementations for LIST_VIEW and LARGE_LIST_VIEW array formats (GH-35344).
Improved casting (GH-39050 and GH-39049).
Support row group filtering for nested paths for C++ and Parquet (GH-39064)
Maintenance: migrated the Gandiva JIT engine to LLJIT (GH-37848).

Reply:
For parquet C++ part:

```
New features: 
* Support row group filtering for nested paths for C++ and Parquet (GH-39064)
* Implement Parquet Float16 logical type (GH-36036)
* Expose sorting_columns in RowGroupMetaData for Parquet files (GH-35331)
* Support decompressing concatenated gzip members (stream) (GH-38271)

API change:
* Move EstimatedBufferedValueBytes from TypedColumnWriter to ColumnWriter (GH-38887)
* Change parquet TypedComparator operation to const methods (GH-38874)
* Remove deprecated AppendRowGroup(int64_t num_rows) (GH-39208)
* Add api to get RecordReader from RowGroupReader (GH-37002)

Bug fixes:
* Add more closed file checks for ParquetFileWriter to Prevent from used-after-close (GH-38390)

Performance enhancement:
* Faster Scalar BYTE_STREAM_SPLIT encoding/decoding (GH-38542)
* Faster reading Parquet FLBA (GH-39124, GH-39413)
* Using bloom_filter_length in parquet 2.10 to optimize bloom filter read (GH-38860)
```

Reply:
For parquet python part:

```
New features: 
* Parquet Support write and validate Page CRC (GH-37242)
```